### PR TITLE
fix(L2): Design tweaks for L2 (Optimism)

### PR DESCRIPTION
**Pull request by callil**
- Rework header responsiveness for network selector UX (no overlaps, minimized style and words)
- - Clean up style consistency of menus
- - Page switcher on bottom, network and account on top
- Switch Optimistic Ethereum -> Optimism for alpha
- Clean up pool page for mobile
-- Remove toggle for checkbox on pools page 
-- Clean up list items design
-- Clean up mobile view (remove CTA)

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -91,19 +90,13 @@ const HeaderElement = styled.div`
...
 
-const HeaderElementWrap = styled.div`
-  display: flex;
-  align-items: center;
-`
-
 const HeaderRow = styled(RowFixed)`
   ${({ theme }) => theme.mediaWidth.upToMedium`
-   width: 100%;
+  //  width: 100%;
```
Comment (justindomingue) [inline src/components/Header/index.tsx:99]: probably better to clean this up

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -63,7 +62,7 @@ const HeaderControls = styled.div`
   align-items: center;
   justify-self: flex-end;
 
-  ${({ theme }) => theme.mediaWidth.upToMedium`
+  /* ${({ theme }) => theme.mediaWidth.upToMedium`
```
Comment (justindomingue) [inline src/components/Header/index.tsx:65]: should we just remove this?

File: src/utils/switchToNetwork.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
-import { SupportedChainId } from 'constants/chains'
+import { SupportedL1ChainId, SupportedL2ChainId } from 'constants/chains'
 import { BigNumber, utils } from 'ethers'
 import { Web3Provider } from '@ethersproject/providers'
 
 interface SwitchNetworkArguments {
   library: Web3Provider
-  chainId: SupportedChainId
+  chainId: SupportedL1ChainId | SupportedL2ChainId
```
Comment (moodysalem) [inline src/utils/switchToNetwork.ts:7]: this change can be undone

File: src/pages/Pool/CTACards.tsx [hunk 4]
```diff
@@ -104,52 +116,41 @@ const HeaderText = styled(TYPE.label)`
...
           <ResponsiveColumn>
             <HeaderText style={{ alignSelf: 'flex-start' }}>
-              <Trans>Top pools</Trans>
+              <Trans>Top pools</Trans> ↗
             </HeaderText>
             <TYPE.body fontWeight={300} style={{ alignSelf: 'flex-start' }}>
               <Trans>Explore popular pools on Uniswap Analytics.</Trans>
             </TYPE.body>
-            <HeaderText style={{ alignSelf: 'flex-end' }}>↗</HeaderText>
+            {/* <HeaderText style={{ alignSelf: 'flex-end' }}>↗</HeaderText> */}
```
Comment (moodysalem) [inline src/pages/Pool/CTACards.tsx:153]: can we delete all the commented out code

File: src/pages/Pool/CTACards.tsx [hunk 5]
```diff
@@ -23,35 +21,49 @@ const CTASection = styled.section`
...
   * {
     color: ${({ theme }) => theme.text1};
     text-decoration: none !important;
   }
 
   :hover {
-    border: 1px solid ${({ theme }) => theme.bg5};
-    background-color: ${({ theme }) => theme.bg2};
+    border: 1px solid ${({ theme }) => theme.bg0};
+    /* background-color: ${({ theme }) => theme.bg2}; */
```
Comment (moodysalem) [inline src/pages/Pool/CTACards.tsx:48]: delete

File: src/components/Toggle/index.tsx [hunk 6]
```diff
@@ -20,9 +20,9 @@ const ToggleElement = styled.span<{ isActive?: boolean; isOnSwitch?: boolean }>`
 
 const StyledToggle = styled.button<{ isActive?: boolean; activeElement?: boolean }>`
   border-radius: 12px;
-  border: 2px solid;
-  border-color: ${({ theme, isActive }) => (isActive ? theme.primary1 : theme.bg3)};
-  background: ${({ theme }) => theme.bg1};
+  border: none;
+  /* border-color: ${({ theme, isActive }) => (isActive ? theme.primary1 : theme.bg3)}; */
```
Comment (moodysalem) [inline src/components/Toggle/index.tsx:24]: delete

**Pull request merged**